pkgs/development/compilers/llvm/12/llvm/default.nix

```diff
@@ -96,6 +96,7 @@ in stdenv.mkDerivation (rec {
     rm test/DebugInfo/X86/convert-debugloc.ll
     rm test/DebugInfo/X86/convert-inlined.ll
     rm test/DebugInfo/X86/convert-linked.ll
+    rm test/DebugInfo/X86/vla-multi.ll
```

Comment:
That is disabling the test for every i383 platform not only armv7l

Reply:
It's under `optionalString stdenv.hostPlatform.isAarch32` so it isn't.